pkg/platform/aws/aws.go

```diff
@@ -180,23 +180,26 @@ func createTerraformConfigFile(cfg *config, terraformRootDir string) error {
 	t := template.New(tmplName)
 	t, err := t.Parse(terraformConfigTmpl)
 	if err != nil {
+		// TODO: Use template.Must().
 		return errors.Wrap(err, "failed to parse template")
 	}
 
 	path := filepath.Join(terraformRootDir, tmplName)
 	f, err := os.Create(path)
 	if err != nil {
-		return errors.Wrapf(err, "failed to create file %q", path)
+		return fmt.Errorf("creating file %q: %w", path, err)
 	}
 	defer f.Close()
 
 	keyListBytes, err := json.Marshal(cfg.SSHPubKeys)
 	if err != nil {
+		// TODO: Render them manually instead of marshalling.
```

Comment:
Can we replace "them" with something concrete to avoid ambiguity?

Reply:
I removed it to be consistent with other TODOs. Now it's just `// TODO: Render manually instead of marshalling.`.